src/jit.rs

```diff
@@ -119,7 +117,7 @@ const CALLEE_SAVED_REGISTERS: [u8; 6] = [
 ];
 
 // Special registers:
-// RDI Pointer to optional typed return value
+// RDI Instruction meter (BPF pc limit)
```

Comment:
pc limit implies a bound on the pc, it's actually a number of instructions executed?

Reply:
It limits the number of instructions, by setting a `pc` limit (which is recalculated at branches to compensate for things like loops and recursion), for details see the documentation comment: https://github.com/solana-labs/rbpf/blob/solana-master/src/jit.rs#L434